deploy-board/deploy_board/webapp/group_view.py

```diff
@@ -30,6 +30,7 @@
 from helpers import autoscaling_metrics_helper
 from diff_match_patch import diff_match_patch
 from deploy_board import settings
+from helpers.exceptions import NotAuthorizedException, TeletraanException
```

Comment:
Is "NotAuthorizedException" used somewhere?

Other changes look good to me.

Reply:
Yes, it is used by other existing functions. Thanks for reminding.

Reply:
looks like NotAuthorizedException is not used in this file.